src/Acheve.TestHost/TestServerExtensions.cs

```diff
@@ -45,7 +49,19 @@ public static class TestServerExtensions
             var validUri = UriDiscover.Discover<TController>(
                 action, tokenValues);
 
-            return server.CreateRequest(validUri.ToLowerInvariant());
+            var requestBuilder = server.CreateRequest(validUri.ToLowerInvariant());
+            
+            var fromBodyArgument = action.ArgumentValues.Values.SingleOrDefault(x => x.IsBody);
```

Comment:
Give the user a option to decide if add or not the content automatically... probably a default paramter in CreateHttpApiRequest is good idea